### PR TITLE
Use AWS profiles

### PR DESCRIPTION
**Pull request by Tonkonozhenko**

### CODE REVIEW

**Comment by Tonkonozhenko** [comment]
@vijaykiran could you review this, please?

File: packages/redshift/sodasql/dialects/redshift_dialect.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ class RedshiftDialect(PostgresDialect):
     def get_aws_credentials_optional(parser: Parser):
         access_key_id = parser.get_str_optional_env('access_key_id')
         role_arn = parser.get_str_optional_env('role_arn')
-        if access_key_id or role_arn:
+        profile_name = parser.get_str_optional_env('profile_name')
```
Comment (vijaykiran) [inline packages/redshift/sodasql/dialects/redshift_dialect.py:29]: This seems to be unused in credentials, I think you missed passing it to AWSCredentials

**Review by vijaykiran — CHANGES_REQUESTED** [review]

**Comment by vijaykiran** [comment]
@Tonkonozhenko thanks again for the PR. Sorry for the delay, I was out of office.

**Comment by vijaykiran** [comment]
https://github.com/sodadata/soda-sql/pull/372

**Pull request closed without merging**